Increase fetch timeout and fix output error

npm run import                                                                                                                                                              ✓  11912  11:35:57
> hacker-news-resolve@1.0.0 import /Users/***/Documents/hacker-news-resolve
> babel-node ./import
Import has been started. Press Crtl+C to stop import
Data importing from news.ycombinator.com [===-----------------] **142/989Fetch timeout (10000 ms)**
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hacker-news-resolve@1.0.0 import: `babel-node ./import`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hacker-news-resolve@1.0.0 import script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/***/.npm/_logs/2017-10-31T08_37_40_174Z-debug.log

--- a/import/api.js
+++ b/import/api.js
@@ -1,4 +1,5 @@
 import fetch from 'isomorphic-fetch'
+import { EOL } from 'os'
 
 const endPoint = 'https://hacker-news.firebaseio.com/v0'
 const timeout = 15000
@@ -21,7 +22,7 @@ const fetchWithRetry = async (url, retry = 0) => {
     ])
 
     if (response === waitResult) {
-      console.error(`\r\nFetch timeout (${timeout} ms)`)
+      console.error(`${EOL}Fetch timeout (${timeout} ms)`)
       process.exit(1)
     }
 

--- a/import/api.js
+++ b/import/api.js
@@ -1,7 +1,7 @@
 import fetch from 'isomorphic-fetch'
 
 const endPoint = 'https://hacker-news.firebaseio.com/v0'
-const timeout = 10000
+const timeout = 15000
 
 const wait = (time, result) =>
   new Promise(resolve => setTimeout(() => resolve(result), time))
@@ -21,7 +21,7 @@ const fetchWithRetry = async (url, retry = 0) => {
     ])
 
     if (response === waitResult) {
-      console.error(`Fetch timeout (${timeout} ms)`)
+      console.error(`\r\nFetch timeout (${timeout} ms)`)
       process.exit(1)
     }
 

--- a/import/index.js
+++ b/import/index.js
@@ -13,5 +13,10 @@ start(
       { width: 20, total }
     )
   },
-  () => bar.tick()
+  () => {
+    bar.tick()
+    if (bar.complete) {
+      console.log('\r\n')
+    }
+  }
 )

--- a/import/index.js
+++ b/import/index.js
@@ -1,4 +1,5 @@
 import ProgressBar from 'progress'
+import { EOL } from 'os'
 
 import { start } from './importer'
 
@@ -16,7 +17,7 @@ start(
   () => {
     bar.tick()
     if (bar.complete) {
-      console.log('\r\n')
+      console.log(EOL)
     }
   }
 )

